<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObjectController</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="controlledLabel">
      <property name="text">
       <string>Controlled objects:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="controlledListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="ComboBoxWithEnter" name="objectsComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>QObject</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QWidget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QFrame</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QPushButton</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QToolButton</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QRadioButton</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCheckBox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QListWidget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QTreeWidget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QTableWidget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QGroupBox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QTabWidget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QComboBox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QFontComboBox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QLineEdit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QTextEdit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QPlainTextEdit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QSpinBox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDoubleSpinBox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QTimeEdit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDateEdit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDateTimeEdit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QSlider</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QLabel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCalendarWidget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QProgressBar</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>Create</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ObjectController" name="objectController" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ObjectController</class>
   <extends>QWidget</extends>
   <header>src/property/objectcontroller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBoxWithEnter</class>
   <extends>QComboBox</extends>
   <header>src/widgets/comboboxwithenter.h</header>
   <slots>
    <signal>enterPressed()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
